lib/src/tag/tagger.dart

```diff
@@ -70,6 +70,7 @@
 /// * platform:windows
 ///
 /// A package has the same platform tags as the primary library.
+library tagger;
```

Comment:
Consider dropping the library names (here and elsewhere):

```suggestion
library;
```